src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1178,7 +1178,7 @@ public class MsmqDequeueStrategy
     }
 }
 
-namespace NServiceBus.Unicast.Transport
+namespace NServiceBus.Unicast.Transports
```

Comment:
Do you want to me revert individual files? Because a merge of

NServiceBus.Transports with NServiceBus.Transports will lead to NServiceBus.Transports ;)